src/account/UpgradeableModularAccount.sol

```diff
@@ -262,6 +262,41 @@ contract UpgradeableModularAccount is
         return returnData;
     }
 
+    /// @inheritdoc IPluginExecutor
+    function executeWithAuthorization(bytes calldata data, bytes calldata authorization)
```

Comment:
Again re: naming (we can punt) - this is still (runtime) validation, so a bit surprising to introduce new vocabulary `authorization`.

Reply:
Yeah happy to revisit. For some background on why I started with this, I saw this field as analogous to the "signature" data in user ops, but we've overloaded that term in too many places already (including 1271), and it's just the authorization for one particular action.